src/pages/home/ReportScreen.js

```diff
@@ -123,9 +125,7 @@ class ReportScreen extends React.Component {
 
     componentDidMount() {
         this.storeCurrentlyViewedReport();
-        if (window.visualViewport) {
-            window.visualViewport.addEventListener('resize', this.viewportOffsetTop);
-        }
+        this.removeViewportResizeListener = addViewportResizeListener(this.viewportOffsetTop);
```

Comment:
Hmm actually this is a pretty bad name for this function. Can we give it a better one while we are here?

Reply:
Maybe I am confused though this looks like a variable and not a function

Reply:
Oh I see it now we did this:

```js
        this.viewportOffsetTop = this.updateViewportOffsetTop.bind(this);
```

yeah that should be 

```js
        this.updateViewportOffsetTop = this.updateViewportOffsetTop.bind(this);
```

Reply:
Yep good point, that's confusing. I'll change it.